packages/cli/src/constructs/check.ts

```diff
@@ -76,10 +76,8 @@ export abstract class Check extends Construct {
   alertChannels?: Array<AlertChannel>
   __checkFilePath?: string // internal variable to filter by check file name from the CLI
 
-  static readonly __checklyType = 'checks'
```

Comment:
Any reason we removed these as well?

Reply:
This was used for registering the construct with the `Project`.

Before, project stored all of the `synthesized()` constructs - so `Project` wouldn't know what type it was receiving. 

Now that we're actually registering a `Construct` object. We don't need the type anymore since we can just check the type with `instanceof`.

https://github.com/checkly/checkly-cli/blob/903f12f6a95ea7b8d2556fe61f50515a6860d6c9/packages/cli/src/constructs/project.ts#L48-L60

Reply:
This works in our case since we have a few types but becomes just too much we have 20-30 different construct classes. This was the reason I added this as a "premature optimization"

Reply:
@clample I think I wanna keep the types. I have some plans with them in the future to add further types to the return values for `synthesize` and https://github.com/checkly/checkly-cli/pull/488/files#diff-f71a863e9534b5c479e26efeae202cafb5f80f992820b15a57d657e2e6a360e6R68-R73

Reply:
I'll try to switch it back to types